script/fetch-issues.js

```diff
@@ -1,33 +1,13 @@
-const fs = require('fs');
-const moment = require('moment');
-const GraphQL = require('graphql-client');
-const sleep = require('sleep');
+import { writeFile } from 'fs';
```

Comment:
This is not necessary, let me roll back.

Reply:
However, 
```javascript
import GraphQLClient from '../src/github-graphql';
```
this is necessary, otherwise there would be error:

`TypeError: GraphQLClient is not a constructor`

Not sure about the reason though...